docs/faq.md

```diff
@@ -1,5 +1,13 @@
 # Frequently Asked Questions
 
+## What are the IP addresses of Cirrus CI?
+
+Cirrus CI control plane uses three IP addresses:
+
+* `34.117.12.6` - IP address for ingress connections to Cirrus CI API and all `*.cirrus-ci.com` domains.
```

Comment:
I don't quite understand how Cirrus CI can *use* a specific IP for ingress connections (but it makes sense for egress connections, though). How about simplifying this to:

```suggestion
* `34.117.12.6` - IP address of the Cirrus CI API and all `*.cirrus-ci.com` domains.
```